### ISSUE TITLE
ConnectV4MappedIPEndPointToV4Host_Success failed in CI on Linux

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#3744]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_linux_debug_tst_prtest/3196/console
```
System.Net.Sockets.Tests.DualMode.ConnectV4MappedIPEndPointToV4Host_Success [FAIL]
16:14:57       System.Net.Sockets.SocketException : Unknown error 10054
16:14:57       Stack Trace:
16:14:57             at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
16:14:57             at System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
16:14:57             at System.Net.Sockets.Tests.DualMode.DualModeConnect_IPEndPointToHost_Helper(IPAddress connectTo, IPAddress listenOn, Boolean dualModeServer)
16:14:57             at System.Net.Sockets.Tests.DualMode.ConnectV4MappedIPEndPointToV4Host_Success()
16:14:57             at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
16:14:57             at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
16:14:57             at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
```

**Comment by stephentoub** [comment]
Failed again here:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_linux_debug_tst_prtest/3285/console

**Comment by ericeil** [comment]
Looks like the same underlying cause as #3494.

**Comment by ericeil** [comment]
Closed with #5907
